shell/deploy-multi-chain.sh

```diff
@@ -288,7 +288,7 @@ for chain in "${provided_chains[@]}"; do
     if [[ ${DETERMINISTIC_DEPLOYMENT} == true ]]; then
         echo -e "${SC}+${NC} Deterministic address"
         if [[ ${sol_script} == "" ]]; then
-            deployment_command+=("script" "script/DeployDeterministicCore.s.sol")
+            deployment_command+=("script" "script/DeployDeterministicProtocol.s.sol")
```

Comment:
> We have discussed this previously that deploying the entire protocol in a single tx would cause gas to exceed block limit.

the problem before was not about gas, but about different optimizer runs between core/periphery which lead to a NFTDescriptor and Dynamic contract too big, now since we use the same foundry configuration it would work (tested):

https://github.com/sablier-labs/v2-core/blob/a34e5287bb77c3e20b8fb9d3ef349a4a80d990aa/foundry.toml#L18

https://github.com/sablier-labs/v2-periphery/blob/2d338a7ff6452b20a25e779f5a960f047a5d16af/foundry.toml#L20

> But lets create another issue for that

don't think we need one as per above reason

Reply:
Oh. I tested it yesterday and it showed gas to be over 30M. How much gas requirement it shows to you?